merge.go

```diff
@@ -0,0 +1,226 @@
+package continuity
+
+import (
+	"log"
+	"path/filepath"
+	"sort"
+	"strings"
+)
+
+// MergeOverlay merges a manifest created from an overlay diff
```

Comment:
How are details of the overlay implementation making it up to the manifest level?